_high_availability/001-Options.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title:  "Options"
+date:   2015-01-27 22:02:36
+categories:
+permalink: /ha/options.html
+---
+
+
+
+Making postgres highly available
+
+When thinking about high availability and postgres, there are two seperate concerns that we need to address. The first one 
+is data replication, that is, how we copy the data to all available nodes and the second one is failover, that is, how we
+detect and manage failure of a node.
+
+This guide deals with the standard PostgreSQL distribution and as such the scenario that covers is that of a single master and 
+multiple slaves. The master serves requests and upon failure, a new master is chosen from the available nodes. 
+
+Replication
+-----------
+
+There are a couple of different options when it comes to replication, with different tradeoffs between them.
```

Comment:
I would probably cut the file/block device based failover from this. That's becoming a "special needs" solution, from what I can tell, whereas streaming replication performs well and is portable to most situations.